k8s/charts/seaweedfs/templates/filer-statefulset.yaml

```diff
@@ -234,6 +234,10 @@ spec:
             - name: data-filer
               mountPath: /data
             {{- end }}
+            - name: notification-config
```

Comment:
conditionally add this?

Reply:
@chrislusf I've updated the implementation.

- The configmap and associated volume mount are now conditionally set.
- In `values.yaml`, `filer.notificationConfig` defaults to empty.
- I've added in `values.yaml` a comment and an example to clarify usage.